src/WSRequestHandler_Sources.cpp

```diff
@@ -315,6 +315,97 @@ RpcResponse WSRequestHandler::SetVolume(const RpcRequest& request)
 	return request.success();
 }
 
+/**
+* Changes whether an audio track is active for a source.
+*
+* @param {String} `source` Source name.
+* @param {int} `track` Audio tracks 1-6.
+* @param {boolean} `active` Whether audio track is active or not.
+*
+* @api requests
+* @name SetTracks
+* @category sources
+* @since 4.0.0
```

Comment:
We use `@since unreleased` for new requests that have not been put into a release yet.